syft/workers/base.py

```diff
@@ -112,7 +112,14 @@ def __init__(
         self.id = id
         self.is_client_worker = is_client_worker
         self.log_msgs = log_msgs
-        self.verbose = verbose
+        if hasattr(hook, "verbose") and hook.verbose is True:
+            self.verbose = True
+        else:
+            self.verbose = verbose
```

Comment:
I guess the verbose option provided in the worker init should be prioritary on the hook.verbose

Reply:
Done!